runtime/binding-kafka-grpc/src/main/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/stream/KafkaGrpcRemoteServerFactory.java

```diff
@@ -598,6 +584,25 @@ private void queueGrpcMessage(
             grpcQueueSlotOffset = queueMessage.limit();
         }
 
+        private void doKafkaCommitOffset(
+            long traceId,
+            long authorization,
+            int partitionId,
+            long partitionOffset)
+        {
+
+            Flyweight commitFlushEx = kafkaFlushExRW
+                .wrap(extBuffer, 0, extBuffer.capacity())
+                .typeId(kafkaTypeId)
+                .merged(m -> m.consumer(mc -> mc
+                    .progress(p -> p
+                        .partitionId(partitionId)
+                        .partitionOffset(partitionOffset + 1L))))
```

Comment:
Please hoist this as a local variable called `nextPartitionOffset` to better document the code.